Sources/Misc/DangerousSettings.swift

```diff
@@ -78,16 +85,19 @@ import Foundation
      * - Parameter autoSyncPurchases: Disable or enable subscribing to the StoreKit queue.
      * If this is disabled, RevenueCat won't observe the StoreKit queue, and it will not sync any purchase
      * automatically.
+     * - Parameter usesStoreKit2JWS: Disable or enable sending StoreKit 2 JWS tokens to RevenueCat
+     * instead of StoreKit 1 receipts.
      */
-    @objc public convenience init(autoSyncPurchases: Bool = true) {
+    @objc public convenience init(autoSyncPurchases: Bool = true, usesStoreKit2JWS: Bool = false) {
```

Comment:
This parameter isn't being used ⚠️

Reply:
I don't think we're actually using this constructor, so let's remove it from the `public` API?